monai/data/sliding_window_inference.py

```diff
@@ -77,22 +81,29 @@ def sliding_window_inference(inputs, roi_size, sw_batch_size, predictor):
     output_classes = output_rois[0].shape[1]
     output_shape = [batch_size, output_classes] + list(image_size)
 
+    # Create importance map
+    importance_map = compute_importance_map(roi_size, mode=blend_mode)
+    importance_map = torch.from_numpy(importance_map).to(device=inputs.device)
```

Comment:
Suggest using [torch.as_tensor()](https://pytorch.org/docs/stable/torch.html#torch.as_tensor) as it's recommented in latest PyTorch programming.
Thanks.

Reply:
Fixed.